clients/fides-js/src/components/tcf/TcfVendors.tsx

```diff
@@ -281,12 +281,16 @@ const PagedVendorData = ({
     otherVendors: VendorRecord[];
   } = useMemo(
     () => ({
-      gvlVendors: activeChunk.filter((v) => v.isGvl),
-      otherVendors: activeChunk.filter((v) => !v.isGvl),
+      gvlVendors: activeChunk?.filter((v) => v.isGvl),
+      otherVendors: activeChunk?.filter((v) => !v.isGvl),
     }),
     [activeChunk]
   );
 
+  if (!activeChunk) {
+    return null;
+  }
+
```

Comment:
handling (unlikely) scenario of having 0 vendors configured for a given experience